plugins/action/synchronize.py

```diff
@@ -205,7 +213,6 @@ def run(self, tmp=None, task_vars=None):
                 "so it cannot work." % self._connection.transport)
             return result
 
-        use_ssh_args = _tmp_args.pop('use_ssh_args', None)
 
```

Comment:
this is the "too many blank lines" pep8 check complains about :)
```suggestion
```
The integration test failing on Mac OS X doesn't seem to be related to this PR.

Reply:
Thanks.